parsing.js

```diff
@@ -96,6 +103,8 @@ function* verseGenerator(document, verses) {
  * @param {string} verse_id
  * @return a verse object that contains the arabic uthmani repr, transliteration... and?
```

Comment:
So this method isn't necessary anymore as we will have the entire Quran stored in our database (i.e. this step is already complete).

Reply:
Feel free to remove it